tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["node_modules", "test", "dist", "**/*spec.ts"]
+  "include": ["src"]
```

Comment:
We still want to exclude `**/*spec.ts` ( or actually `**/*.spec`) for the build.

Reply:
Yes you are right about this. Thanks for the feedback.